source/extensions/filters/http/cache/inline_headers_handles.h

```diff
@@ -9,46 +9,93 @@ namespace Extensions {
 namespace HttpFilters {
 namespace Cache {
 
-// Request headers inline handles
 inline Http::RegisterCustomInlineHeader<Http::CustomInlineHeaderRegistry::Type::RequestHeaders>
-    authorization_handle(Http::CustomHeaders::get().Authorization);
+    authorization_handle() {
```

Comment:
I think you'd need to split this into .h and .cc files to make it work.

Reply:
agreed; also split into h/cc so the lazy init happens in the cc.